### PR TITLE
Version 3.3.5

### LINKED ISSUE #130: Wrong tag for cdn inclusion in README file
### Describe the bug

In Readme file, we are instructed to use `uploadcare_widget_tag`, which appears to be undefined. By searching in the code, the correct tag to use is `uploadcare_include_tag`. Please, update the instructions accordingly.

### Environment

 - Library version: 3.3.4
 - Language/framework version: Ruby 3.0.2, Rails 7.0.1
 - OS version: Debian 11.6

image

### PR DESCRIPTION
**Pull request by kraft001**
## Description

Fixes #130 

## Checklist

- [x] Tests (if applicable)
- [x] Documentation (if applicable)
- [x] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based now on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 3.3.5 — 2023-10-15
+
+### Fixed
+
+* Documentation issue with `uploadcare_include_tag`
```
Comment (rsedykh) [inline CHANGELOG.md:11]: Plus some maintenance?

Comment (vipulnsward) [reply CHANGELOG.md:11]: Done

**Review by rsedykh — APPROVED** [review]

**Comment by rsedykh** [comment]
Though I'm not sure if version dropping isn't bumping minor version, e.g. 3.4?

**Comment by vipulnsward** [comment]
Correct, updated in https://github.com/uploadcare/uploadcare-rails/pull/132

Closing this one.

**Pull request closed without merging**